www/include/configuration/configObject/host/formHost.php

```diff
@@ -445,6 +430,11 @@ function allInSameInstance($hosts, $instanceId)
 );
 $form->addElement('static', 'tplText', _("Using a Template allows you to have multi-level Template connection"));
 
+$tplDeRoute = './include/common/webServices/rest/internal.php?object=centreon_configuration_hosttemplate'
```

Comment:
what means $tplDeRoute ? -> can you choose a more explicit variable name
caution -> change line 435 too

Reply:
I saw "$timezDeRoute" at line 391 so I tried to be as consistent as possible. Will change it.